src/read.c

```diff
@@ -3599,7 +3615,11 @@ static avifResult avifCodecCreateInternal(avifCodecChoice choice, const avifTile
 
 static avifBool avifTilesCanBeDecodedWithSameCodecInstance(avifDecoderData * data)
 {
-    if (data->color.tileCount == 1 && (data->alpha.tileCount == 1 || data->gainMap.tileCount == 1)) {
+    avifBool hasSingleTileAlphaOrGainMap = data->alpha.tileCount == 1;
+#if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
+    hasSingleTileAlphaOrGainMap = hasSingleTileAlphaOrGainMap || data->gainMap.tileCount == 1;
+#endif
+    if (data->color.tileCount == 1 && hasSingleTileAlphaOrGainMap) {
```

Comment:
I suggest two changes to the comments below.

Line 3623: alpha or gain map plane => alpha plane or gain map

Line 3624: alpha (or gain map) plane => alpha plane (or gain map)